src/test/java/utilities/Util.java

```diff
@@ -1,7 +1,20 @@
 package utilities;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
 public class Util {
 
+	
+	/*
+	 * Clicks on buttons based on xpath of button 
+	 * @Author: Jack Vu 
+	 * 
+	 */
+	public static void buttonClickFunction(WebDriver driver, String xpathOfButton) {
```

Comment:
It would be a better idea to use `By` object instead of xpath, would be more reusable

Reply:
Used By object, should be ready to go.